### PR TITLE
Expanded Okta AS, Custom AS doc 

### PR DESCRIPTION
**Pull request by mystiberry-okta**
## Description:
- Beefing up the description of org AS vs. custom AS. I coined short names for them. Don't hate me.

## Type of Pull Request:

- [x] Content (Documentation updates or typo-fixes)
- [ ] Blog Post
- [ ] Functional (CSS changes, JS updates, or layout modifications)

### Resolves:
* [OKTA-110671](https://oktainc.atlassian.net/browse/OKTA-110671)

## How Has This Been Tested?
- [x] I have built this locally and verified that it does not break existing functionality.
- [ ] For functional changes, I have tested against Chrome, Firefox, and Safari, and mobile.
- [ ] For functional changes, I have added tests.

## Primary Reviewers:
@cwu-okta @johngronberg-okta anyone welcome to review!

### CODE REVIEW

**Comment by mystiberry-okta** [comment]
Everyone is welcome to look at the new descriptions for org AS vs. custom AS.

File: _source/_standards/OAuth/index.md [hunk 1]
```diff
@@ -315,19 +315,44 @@ The actions in a rule define the lifetime of the Access Token and Refresh Token.
 
 ## Authorization Servers
 
-API Access Management allows you to build custom Authorization Servers in Okta which can be used to protect your own API endpoints. An Authorization Server defines your security boundary, for example “staging” or “production.” Within each Authorization Server you can define your own OAuth scopes, claims, and access policies. This allows your apps and your APIs to anchor to a central authorization point and leverage the rich identity features of Okta, such as Universal Directory for transforming attributes, adaptive MFA for end-users, analytics, and system log, and extend it out to the API economy.
+At its core, an authorization server is simply an OAuth 2.0 token minting engine.
```
Comment (richardmateosian-okta) [inline _source/_standards/OAuth/index.md]: How about
"The core of an authorization server is an engine for minting OAuth 2.0 tokens."?

File: _source/_standards/OAuth/index.md [hunk 2]
```diff
@@ -315,19 +315,44 @@ The actions in a rule define the lifetime of the Access Token and Refresh Token.
...
+| Hosted by Okta                 | &#10004;             | &#10004;  |
+| Manage identity in Okta        | &#10004; recommended | &#10004;  |
+| Add groups claim               | &#10004;             | &#10004;  |
+| Add user-profile attributes    | &#10004;             | &#10004;  |
+| Manage resources outside Okta  |                      | &#10004;  |
+| Requires API Access Management |                      | &#10004;  |
+| Org Admin creates one or more  |                      | &#10004;  |
+
+A Custom AS defines your security boundary, for example “staging” or “production.” Within each authorization server you can define your own OAuth scopes, claims, and access policies.
+This allows your apps and your APIs to anchor to a central authorization point and leverage the rich identity features of Okta, such as Universal Directory for transforming attributes, adaptive MFA for end-users, analytics, and system log, and extend it out to the API economy.
```
Comment (richardmateosian-okta) [inline _source/_standards/OAuth/index.md]: extend **them** out . . .

File: _source/_standards/OAuth/index.md [hunk 3]
```diff
@@ -315,19 +315,44 @@ The actions in a rule define the lifetime of the Access Token and Refresh Token.
...
+Each Custom AS has a unique issuer URI and its own signing key for tokens in order to keep proper boundary between security domains.
```
Comment (richardmateosian-okta) [inline _source/_standards/OAuth/index.md]: "a proper boundary" or "proper boundaries"

File: _source/_standards/OAuth/index.md [hunk 4]
```diff
@@ -315,19 +315,44 @@ The actions in a rule define the lifetime of the Access Token and Refresh Token.
...
+
+The Custom AS can also act as an OpenID Connect Provider, which means you can request ID Tokens in addition to Access Tokens from the authorization server endpoints.
+
+To configure a Custom AS, log into your org and navigate to **Security** > **API** > **Add Authorization Server**.
```
Comment (richardmateosian-okta) [inline _source/_standards/OAuth/index.md]: Do we say "log into"? Isn't our standard "sign on to"?

File: _source/_use_cases/api_access_management/index.md [hunk 5]
```diff
@@ -37,7 +37,7 @@ See [OAuth 2.0 and OpenID Connect](#oauth-20-and-openid-connect) for details.
 Centralizing the management of your APIs makes it easier for others to consume your API resources.
 Using Okta's OAuth-as-a-Service feature, API Access Management, provides many benefits:
 
-* Create one or more hosted authentication servers, which makes it easier to manage sets of API access for multiple client apps across many customer types.
+* Create one or more [Custom Authorization Servers (Custom AS)](/standards/OAuth/index.html#authorization-servers), hosted on Okta. A Custom AS makes it easier to manage sets of API access for multiple client apps across many customer types.
```
Comment (richardmateosian-okta) [inline _source/_use_cases/api_access_management/index.md]: "sets" and "multiple" are probably not both needed. I suggest dropping "sets." "API access" needs something to complete it -- API access methods, for example.

File: _source/_standards/OAuth/index.md (hunk 1 above)
Comment (mystiberry-okta) [inline _source/_standards/OAuth/index.md]: The subject of the sentence isn't the AS core, that's why the phrase is set off.

Comment (mystiberry-okta) [inline _source/_standards/OAuth/index.md]: Would "You can think of an AS as an OAuth 2.0 token minting engine" be better?

File: _source/_standards/OAuth/index.md (hunk 2 above)
Comment (mystiberry-okta) [inline _source/_standards/OAuth/index.md]: good catch!

File: _source/_standards/OAuth/index.md (hunk 4 above)
Comment (mystiberry-okta) [inline _source/_standards/OAuth/index.md]: will check--not sure we have a standard yet.

File: _source/_docs/api/resources/oidc.md [hunk 6]
```diff
@@ -723,6 +723,8 @@ This is a starting point for OpenID Connect flows such as implicit and authoriza
 authenticates the user and returns tokens along with an authorization grant to the client application as a part
 of the response.
 
+> Because this endpoint works with the [Org AS](/standards/OAuth/index.html#authorization-servers), you don't need an authorization server ID.
```
Comment (royalchan-okta) [inline _source/_docs/api/resources/oidc.md]: Does this need to be mentioned at the /token endpoint as well?

File: _source/_use_cases/api_access_management/index.md (hunk 5 above)
Comment (mystiberry-okta) [inline _source/_use_cases/api_access_management/index.md]: no, we really mean bundle1 for apps x,y,z but bundle2 for a,b,c. The product name is API Access Management. So there is such a thing as API access. Okta provides a way (custom AS) to say who gets what access (via API) to which resources to do things with which clients. If that's not clear from the sentence, it would help me to know what wasn't clear. Thanks!

File: _source/_standards/OAuth/index.md (hunk 4 above)
Comment (mystiberry-okta) [inline _source/_standards/OAuth/index.md]: sign in is the standard, great catch!

File: _source/_standards/OAuth/index.md (hunk 3 above)
Comment (mystiberry-okta) [inline _source/_standards/OAuth/index.md]: ack.

File: _source/_standards/OAuth/index.md (hunk 1 above)
Comment (richardmateosian-okta) [inline _source/_standards/OAuth/index.md]: My problems with the original were:
 - the word "simply"
 - the hard to parse phrase "OAuth 2.0 token minting engine" 

If there is no actual core that functions as an engine, "you can think of" is better.

File: _source/_use_cases/api_access_management/index.md (hunk 5 above)
Comment (richardmateosian-okta) [inline _source/_use_cases/api_access_management/index.md]: If you mean "sets of bundles," that's what I stumbled on. I didn't get that there is a hierarchy.

Comment (richardmateosian-okta) [inline _source/_use_cases/api_access_management/index.md]: I'm not sure you can pull "API access" out of "API Access Management" and treat it as a stand-alone term. It feels incomplete to me.

File: _source/_standards/OAuth/index.md (hunk 1 above)
Comment (mystiberry-okta) [inline _source/_standards/OAuth/index.md]: Yah, 'simply' set off my adverb radar too, but I think it's warranted in this conceptual context--it pairs with "at its core". I'll go with 
At its core, an AS is simply an engine for minting OAuth 2.0 tokens.

Great catch, I wasn't noticing the 4 word NP, one of my pet peeves :)

File: _source/_use_cases/api_access_management/index.md (hunk 5 above)
Comment (mystiberry-okta) [inline _source/_use_cases/api_access_management/index.md]: https://apigee.com/about/cp/what-is-api-access
http://creators.ning.com/forum/topics/what-is-api-access
https://support.google.com/a/answer/60757
http://about.crunchbase.com/knowledge-center/api-access/

I'll add it to the style guide--I didn't realize it was a common industry term (just thought it was a regular old part of English, like VIP Access, unauthorized access, etc.).

File: _source/_docs/api/resources/oidc.md (hunk 6 above)
Comment (mystiberry-okta) [inline _source/_docs/api/resources/oidc.md]: Yes! Great catch!

**Comment by mystiberry-okta** [comment]
Rebased to make new reviews easy.

File: _source/_docs/api/resources/oauth2.md [hunk 7]
```diff
@@ -7,15 +7,15 @@ weight: 4
...
+Okta is a fully standards-compliant [OAuth 2.0](http://oauth.net/documentation) authorization server and a certified [OpenID Provider](http://openid.net/certification).
 
 The OAuth 2.0 API provides API security via scoped access tokens, and OpenID Connect provides user authentication and an SSO layer which is lighter and easier to use than SAML.
 
 Explore the OAuth 2.0 API: [![Run in Postman](https://run.pstmn.io/button.svg)](https://app.getpostman.com/run-collection/e4d286b1af2294bb14a0)
 
 ## Endpoints
 
-You can perform authorization and token operations, as well as create, configure, and delete Authorization Servers, policies, rules, scopes, and claims.
+You can perform authorization and token operations, as well as create, configure, and delete Custom AS, policies, rules, scopes, or claims.
```
Comment (jakubvul) [inline _source/_docs/api/resources/oauth2.md]: I would argue that this acronym is unnecessary.

But, if we really have to use it, then we should at least define it somewhere.

But really I don't think we should use it.

File: _source/_docs/api/resources/authn.md [hunk 8]
```diff
@@ -1,6 +1,6 @@
 ---
 layout: docs_page
-title: Authentication API
+title: Authentication
```
Comment (jakubvul) [inline _source/_docs/api/resources/authn.md]: This change requires a chance in the test found out `tests/selenium/spec/side-bar-spec.js`. On line 48, `expectedSubLinks`, you need to change `Authentication API` to `Authentication`. This fixes one of the failing tests.

**Comment by jakubvul-okta** [comment]
I would also add (using another account, oops) that the "AS" acronym is also very unsearchable itself, both via Google and in the page itself.

File: _source/_docs/api/resources/authn.md (hunk 8 above)
Comment (mystiberry-okta) [inline _source/_docs/api/resources/authn.md]: Thanks!

**Comment by mystiberry-okta** [comment]
I agree with @jakubvul about the short name--going to get rid of it unless anyone objects. It does mean we want to be careful when talking about servers, to avoid the dreaded 4- or 5-word noun phrase (very hard to read), since this product name already has 3 words. Searchability beats scannability/velocity.

File: _source/_docs/api/resources/authn.md (hunk 8 above)
Comment (mystiberry-okta) [inline _source/_docs/api/resources/authn.md]: @vijetmahabaleshwar-okta or @rchild-okta we really shouldn't have tests hardwired to specific content, should we?

Comment (mystiberry-okta) [inline _source/_docs/api/resources/authn.md]: Oops, just saw Vijet's email, this is known issue, and I'll fix the test.

**Comment by mystiberry-okta** [comment]
This is going to miss the 2017.27 release.

**Comment by mystiberry-okta** [comment]
The whole Federation team is optional, but @johngronberg-okta I need your review. Let me know if I haven't captured the distinction between platform use cases and IT use cases with sufficient crispness.

File: _source/_standards/OAuth/index.md [hunk 9]
```diff
@@ -331,19 +331,53 @@ The actions in a rule define the lifetime of the Access Token and Refresh Token.
...
 
-At its core, an Authorization Server is simply an OAuth 2.0 token minting engine.
-Each Authorization Server has a unique issuer URI and its own signing key for tokens in order to keep proper boundary between security domains.
-The Authorization Server also acts as an OpenID Connect Provider, which means you can request ID tokens in addition to access tokens from the Authorization Server endpoints.
-To configure an Authorization Server, log into your org and navigate to **Security** > **API** > **Add Authorization Server**.
+Okta provides two types of authorization servers:
+
+* Okta Authorization Server:
+Use the Okta Authorization Server to perform single-sign on with Okta or log users into their apps displayed on the Okta home page.
+Okta hosts and manages Okta Authorization Server. It can't be configured,
```
Comment (johngronberg-okta) [inline _source/_standards/OAuth/index.md]: "Okta hosts and manages the Okta Authorization Server"

File: _source/_standards/OAuth/index.md [hunk 10]
```diff
@@ -331,19 +331,53 @@ The actions in a rule define the lifetime of the Access Token and Refresh Token.
...
+though you can add a [groups claim](/docs/api/resources/oidc.html#scope-dependent-claims-not-always-returned)
+and [app-user profile attributes](/docs/api/resources/apps.html#application-user-properties) to a client.
+The Access Token minted by Okta Authorization Server is consumed by Okta APIs. Its scope is always Okta specific and can't be validated by your applications or APIs.
```
Comment (johngronberg-okta) [inline _source/_standards/OAuth/index.md]: The Access Token minted by the Okta Authorization Server

Comment (johngronberg-okta) [inline _source/_standards/OAuth/index.md]: also the use of 'scope' here is a bit confusing since scope means something different in the OIDC context. Audience would be more appropriate.

File: _source/_standards/OAuth/index.md [hunk 11]
```diff
@@ -331,19 +331,53 @@ The actions in a rule define the lifetime of the Access Token and Refresh Token.
...
+
+    >Note: The Okta Authorization Server is available without any additional features enabled.
+
+* Custom Authorization Server:
+Use a Custom Authorization Server to secure your APIs. Okta hosts one or more Custom Authorization Servers that the Org Admin creates
+and configures, using the Okta User Interface or Okta API. The Access Token minted by a Custom Authorization Server is consumed by your APIs. 
+You can specify the audience to make sure that the Access Token is for your APIs.
+Scopes can be modified to support custom authorization in your application. 
+
+     >Note: Custom Authorization Server is available as part of the API Access Management feature.
```
Comment (johngronberg-okta) [inline _source/_standards/OAuth/index.md]: "Custom Authorization Servers are..."

File: _source/_standards/OAuth/index.md [hunk 12]
```diff
@@ -331,19 +331,53 @@ The actions in a rule define the lifetime of the Access Token and Refresh Token.
...
+| Org Admin creates one or more            |                           | &#10004;                    |
+| Validate the Access Token in custom code |                           | &#10004;                    |
+| Custom Scopes                            |                           | &#10004;                    |
+| Custom Claims                            |                           | &#10004;                    |
+| Custom Access Policies and Rules         |                           | &#10004;                    |
+
+
+To configure a Custom Authorization Server, sign in to your org and navigate to **Security** > **API** > **Add Authorization Server**.
+
+We recommend that if you use Custom Authorization Server, that you use it instead of the Okta Authorization Server for
```
Comment (johngronberg-okta) [inline _source/_standards/OAuth/index.md]: if you use a Custom Authorization Server

File: _source/_use_cases/api_access_management/index.md [hunk 13]
```diff
@@ -37,7 +37,7 @@ See [OAuth 2.0 and OpenID Connect](#oauth-20-and-openid-connect) for details.
 Centralizing the management of your APIs makes it easier for others to consume your API resources.
 Using Okta's OAuth-as-a-Service feature, API Access Management, provides many benefits:
 
-* Create one or more hosted authentication servers, which makes it easier to manage sets of API access for multiple client apps across many customer types.
+* Create one or more [Custom Authorization Servers](/standards/OAuth/index.html#authorization-servers), hosted on Okta. Custom Authorization Server makes it easier to manage sets of API access for multiple client apps across many customer types.
```
Comment (johngronberg-okta) [inline _source/_use_cases/api_access_management/index.md]: "Custom Authorization Servers make it..."

**Comment by johngronberg-okta** [comment]
@mystiberry-okta I suggested a few minor changes. Mostly adding "the" or "a" when referencing "a Custom Authorization Server"; perhaps that's just a convention difference, but the text reads awkwardly to me without definite articles.

**Comment by cwu-okta** [comment]
@federations-okta  (now that the alias works again!)

File: _source/_standards/OAuth/index.md (hunk 11 above)
Comment (cwu-okta) [inline _source/_standards/OAuth/index.md]: Question: Do we need the EA liquid tag here? I don't feel strongly about it but wanted to ask if we need to indicate API AM is currently EA.

File: _source/_standards/OAuth/index.md [hunk 14]
```diff
@@ -331,19 +331,53 @@ The actions in a rule define the lifetime of the Access Token and Refresh Token.
...
+Use the Okta Authorization Server to perform single-sign on with Okta or log users into their apps displayed on the Okta home page.
+Okta hosts and manages Okta Authorization Server. It can't be configured,
+though you can add a [groups claim](/docs/api/resources/oidc.html#scope-dependent-claims-not-always-returned)
+and [app-user profile attributes](/docs/api/resources/apps.html#application-user-properties) to a client.
+The Access Token minted by Okta Authorization Server is consumed by Okta APIs. Its scope is always Okta specific and can't be validated by your applications or APIs.
+
+    >Note: The Okta Authorization Server is available without any additional features enabled.
+
+* Custom Authorization Server:
+Use a Custom Authorization Server to secure your APIs. Okta hosts one or more Custom Authorization Servers that the Org Admin creates
```
Comment (cwu-okta) [inline _source/_standards/OAuth/index.md]: Is Org Admin well understood in this context? I ask because we have various roles within an Org (Super Org Admin, Org Admin, Read Only admin, etc.) I understand you are trying to convey this is an administrator who can configure the values but want to ask if Org Admin is well understood for that meaning or if it might be mistaken for our actual Role names.

**Comment by mystiberry-okta** [comment]
Need to clean up the commits after weekly release.

File: _source/_standards/OAuth/index.md (hunk 9 above)
Comment (mystiberry-okta) [inline _source/_standards/OAuth/index.md]: It's a funny thing--once we capitalize stuff, we're supposed to treat these as proper nouns (Okta hosts and managed the Bob would be weird), but I'm finding that really hard to do without sounding crazy. That always happens when a proper noun is also a common noun. Guess we'll go for consistently treating it like a common noun.

File: _source/_standards/OAuth/index.md (hunk 11 above)
Comment (mystiberry-okta) [inline _source/_standards/OAuth/index.md]: I had a question about that. If it goes GA with the preview release, but stays GA in preview only for a month, what are we going to do about the tags for that month? I'll add a link to the API AM topic here, which has the EA tag.

Comment (mystiberry-okta) [inline _source/_standards/OAuth/index.md]: Well, it does now.

Comment (cwu-okta) [inline _source/_standards/OAuth/index.md]: @mystiberry-okta API AM does not have a GA date at the moment. We are GAing OIDC in preview this month, then prod the following month

File: _source/_standards/OAuth/index.md (hunk 14 above)
Comment (mystiberry-okta) [inline _source/_standards/OAuth/index.md]: the actual role name is `ORG_ADMIN`, but how about this to make it super clear:

Custom Authorization Servers are hosted on Okta, and created by a user with the correct permissions (Org Admin or Super Org Admin). You can create Custom Authorization Servers using the Okta Admin UI or the OAuth 2.0 API.

Comment (cwu-okta) [inline _source/_standards/OAuth/index.md]: Since may choose to create a separate role for this in the future can we future proof the wording? How about

```Custom Authorization Servers are hosted on Okta, created and configured by an administrator. You can create Custom Authorization Servers using the Okta Admin UI or the OAuth 2.0 API.```

Comment (cwu-okta) [inline _source/_standards/OAuth/index.md]: Also we should link OAuth 2.0 API to the AS CRUD apis

Comment (mystiberry-okta) [inline _source/_standards/OAuth/index.md]: I made these last two changes in Doc pr 1021 (too much back-and-forth).

File: _source/_standards/OAuth/index.md [hunk 15]
```diff
@@ -330,19 +330,55 @@ The actions in a rule define the lifetime of the Access Token and Refresh Token.
...
+and [app-user profile attributes](/docs/api/resources/apps.html#application-user-properties) to a client.
+The Access Token minted by the Okta Authorization Server is consumed by Okta APIs. Its audience is always Okta specific and can't be validated by your applications or APIs.
+
+    >Note: The Okta Authorization Server is available without any additional features enabled.
+
+* Custom Authorization Server:
+Use a Custom Authorization Server to secure your APIs.
+Custom Authorization Servers are hosted on Okta, and created by a user with the correct permissions (Org Admin or Super Org Admin).
+You can create and configure Custom Authorization Servers using the Okta Admin UI or the OAuth 2.0 API.
+and configures, using the Okta User Interface or Okta API. The Access Token minted by a Custom Authorization Server is consumed by your APIs.
```
Comment (larsjohansen-okta) [inline _source/_standards/OAuth/index.md]: This seems to be a strange way to start a sentence, is there some copy/pasta issue here?

Comment (mystiberry-okta) [inline _source/_standards/OAuth/index.md]: Yup,  I deleted.

**Comment by mystiberry-okta** [comment]
This touches some of the same files as https://github.com/okta/okta.github.io/pull/1021, so I'm going to get that one merged before asking for a further review, as I'll need to merge/resolve conflicts on at least two files.

File: _source/_docs/api/resources/oauth2.md [hunk 16]
```diff
@@ -1,13 +1,13 @@
...
-title: OAuth 2.0 API
+title: OAuth 2.0
 weight: 4
 ---
 # OAuth 2.0 API
 
 {% api_lifecycle ea %}
 
-Okta is a fully standards-compliant [OAuth 2.0](http://oauth.net/documentation) Authorization Server and a certified [OpenID Provider](http://openid.net/certification).
+Okta is a fully standards-compliant [OAuth 2.0](http://oauth.net/documentation) authorization server and a certified [OpenID Provider](http://openid.net/certification).
```
Comment (yuliu-okta) [inline _source/_docs/api/resources/oauth2.md]: I'd remove "fully" since we have not implemented all the specs listed in the link.

File: _source/_docs/api/resources/oauth2.md [hunk 17]
```diff
@@ -18,7 +18,7 @@ To understand more about OAuth 2.0 and Okta:
 
 ## Endpoints
 
-You can perform authorization and token operations, as well as create, configure, and delete Authorization Servers, policies, rules, scopes, and claims.
+You can perform authorization and token operations, as well as create, configure, and delete Custom Authorization Servers, policies, rules, scopes, or claims.
```
Comment (yuliu-okta) [inline _source/_docs/api/resources/oauth2.md]: create, update, and delete

File: _source/_docs/api/resources/oauth2.md [hunk 18]
```diff
@@ -1765,7 +1767,7 @@ curl -v -X GET \
 
 {% api_operation post /api/v1/authorizationServers/:authorizationServerId/credentials/lifecycle/keyRotate %}
 
-Rotates the current keys for the Authorization Server. If you rotate keys, the `ACTIVE` key becomes the `EXPIRED` key, the `NEXT` key becomes the `ACTIVE` key, and the Authorization Server immediately issues tokens signed with the new active key.
+Rotates the current keys for a Custom Authorization Server. If you rotate keys, the `ACTIVE` key becomes the `EXPIRED` key, the `NEXT` key becomes the `ACTIVE` key, and a Custom Authorization Server immediately issues tokens signed with the new active key.
```
Comment (yuliu-okta) [inline _source/_docs/api/resources/oauth2.md]: and the Custom Authorization Server immediately issues tokens ...

File: _source/documentation/stormpath-import.md [hunk 19]
```diff
@@ -189,7 +189,7 @@ The import tool works by iterating over your Stormpath data and then uses the Ok
 Stormpath Resource | Okta Equivalent
 --- | ---
 Account | [User](/docs/api/resources/users.html#user-properties)
-Application | [OAuth 2.0 Client Application](/docs/api/resources/oauth-clients.html) acting as a client for an [OAuth 2.0 Authorization Server](/docs/api/resources/oauth2.html#authorization-operations)
+Application | [Dynamic Client Registration API](/docs/api/resources/oauth-clients.html) acting as a client for an [OAuth 2.0 Authorization Server](/docs/api/resources/oauth2.html#authorization-operations)
```
Comment (jakubvul-okta) [inline _source/documentation/stormpath-import.md]: This doesn't seem right - we're not talking about API equivalents, just the actual entities themselves. I can see that the name of the API has changed, but the underlying entity is still the same right?

File: _source/_docs/api/resources/oidc.md [hunk 20]
```diff
@@ -727,6 +729,8 @@ This is a starting point for OpenID Connect flows such as implicit and authoriza
 authenticates the user and returns tokens along with an authorization grant to the client application as a part
 of the response.
 
+> Because this endpoint works with the [Okta AS](/standards/OAuth/index.html#authorization-servers), you don't need an authorization server ID.
```
Comment (larsjohansen-okta) [inline _source/_docs/api/resources/oidc.md]: do we introduce the term Okta AS somewhere before this point? We might want to stick with Okta Authorization Server if not?

File: _source/_docs/api/resources/oidc.md [hunk 21]
```diff
@@ -898,6 +902,8 @@ https://www.example.com/#error=invalid_scope&error_description=The+requested+sco
 
 The API returns Access Tokens, ID Tokens, and Refresh Tokens, depending on the request parameters. 
 
+>Because this endpoint works with the [Okta Authorization Server](/standards/OAuth/index.html#authorization-servers), you don't need an authorization server ID.
```
Comment (larsjohansen-okta) [inline _source/_docs/api/resources/oidc.md]: We should be consistent with using Okta Authorization Server vs Okta AS.

File: _source/_docs/api/resources/oidc.md (hunk 20 above)
Comment (mystiberry-okta) [inline _source/_docs/api/resources/oidc.md]: Oops, missed that one, thank you!

File: _source/_docs/api/resources/oidc.md (hunk 21 above)
Comment (mystiberry-okta) [inline _source/_docs/api/resources/oidc.md]: Yes, AS shouldn't appear anywhere.

File: _source/documentation/stormpath-import.md (hunk 19 above)
Comment (mystiberry-okta) [inline _source/documentation/stormpath-import.md]: @jakubvul  I just changed the name. Whom should I follow up with about the content?
